WildFly_26.1.0.Final/jakarta-full-platform.adoc

```diff
@@ -0,0 +1,627 @@
+
+Test name: jstl
+[javatest.batch] Completed running 541 tests.
+[javatest.batch] Number of Tests Passed      = 541
+[javatest.batch] Number of Tests Failed      = 0
+[javatest.batch] Number of Tests with Errors = 0
+
+Test name: jta
+[javatest.batch] Completed running 195 tests.
+[javatest.batch] Number of Tests Passed      = 195
+[javatest.batch] Number of Tests Failed      = 0
+[javatest.batch] Number of Tests with Errors = 0
+
+Test name: rmiiiop
+[javatest.batch] Completed running 129 tests.
+[javatest.batch] Number of Tests Passed      = 129
+[javatest.batch] Number of Tests Failed      = 0
+[javatest.batch] Number of Tests with Errors = 0
+
+Test name: securityapi
+[javatest.batch] Completed running 83 tests.
+[javatest.batch] Number of Tests Passed      = 83
+[javatest.batch] Number of Tests Failed      = 0
+[javatest.batch] Number of Tests with Errors = 0
+
+Test name: servlet
+[javatest.batch] Completed running 1725 tests.
+[javatest.batch] Number of Tests Passed      = 1725
+[javatest.batch] Number of Tests Failed      = 0
+[javatest.batch] Number of Tests with Errors = 0
+
+Test name: signaturetest_javaee
+[javatest.batch] Completed running 5 tests.
+[javatest.batch] Number of Tests Passed      = 5
+[javatest.batch] Number of Tests Failed      = 0
+[javatest.batch] Number of Tests with Errors = 0
+
+Test name: webservices12
+[javatest.batch] Completed running 242 tests.
+[javatest.batch] Number of Tests Passed      = 242
+[javatest.batch] Number of Tests Failed      = 0
+[javatest.batch] Number of Tests with Errors = 0
+
+Test name: webservices13
+[javatest.batch] Completed running 53 tests.
+[javatest.batch] Number of Tests Passed      = 53
+[javatest.batch] Number of Tests Failed      = 0
+[javatest.batch] Number of Tests with Errors = 0
+
+Test name: websocket
+[javatest.batch] Completed running 745 tests.
+[javatest.batch] Number of Tests Passed      = 745
+[javatest.batch] Number of Tests Failed      = 0
+[javatest.batch] Number of Tests with Errors = 0
+
+Test name: xa
+[javatest.batch] Completed running 66 tests.
+[javatest.batch] Number of Tests Passed      = 66
+[javatest.batch] Number of Tests Failed      = 0
+[javatest.batch] Number of Tests with Errors = 0
+----
+
+=== Additional standalone TCK Test results
+Standalone test results:
+
+Jakarta Dependency Injection 1.0 TCK
+
+Download URL & SHA-256:
+https://download.eclipse.org/jakartaee/dependency-injection/1.0/jakarta.inject-tck-1.0-bin.zip[jakarta.inject-tck-1.0-bin.zip, b679bac9b1057df894753892a880ba6ade530607dd811157106ed767aa26481f ]
+
+TCK result summary:
+----
+Tests run: 50, Failures: 0, Errors: 0, Skipped: 0
+----
+
+Jakarta Contexts and Dependency Injection 2.0 TCK
+
+Download URL & SHA-256
+https://download.eclipse.org/jakartaee/cdi/2.0/cdi-tck-2.0.6-dist.zip[cdi-tck-2.0.6-dist.zip, 75e969a7a3b3c77332154a2008309aad821a923d8684139242048a7640762808 ]
+
+TCK result summary:
+----
+Tests run: 1807, Failures: 0, Errors: 0, Skipped: 0
```

Comment:
Same comment here; needs a line with '----'

Reply:
Thanks for noticing!  Fixed!